fs/artifact_fetcher.go

```diff
@@ -68,7 +67,7 @@ func newArtifactFetcher(refspec reference.Spec, localStore, remoteStore content.
 	}, nil
 }
 
-func newRemoteStore(refspec reference.Spec) (content.Storage, error) {
+func newRemoteStore(refspec reference.Spec) (*remote.Repository, error) {
```

Comment:
why the change in return type?

Reply:
It is used to [resolve the image ref](https://github.com/awslabs/soci-snapshotter/pull/65/files#diff-c7b54bb28e1654d0ca47364a2c97456c124ee0d3a65854db18d116c509d89136R232) in `fs.go`. 

Returning `content.Storage` would have meant doing something like 

```
repo, _ := remoteStore.(*remote.Repository)
```

I wanted to avoid that.